cupyx/jit/_interface.py

```diff
@@ -81,6 +84,10 @@ def __getitem__(self, grid_and_block):
             block = (block, 1, 1)
         return lambda *args, **kwargs: self(grid, block, args, **kwargs)
 
+    @property
+    def codes(self):
```

Comment:
* I prefer returning it as dict (like `self._cache`) instead of list.
* Let's make it explicit that this is an interface for debugging purposes. How about naming it `cached_codes` instead?

Reply:
Fixed!